infrastructure/modules/psql/variables.tf

```
@@ -0,0 +1,315 @@
+variable "psql_SubnetCidr" {
```

Keep to one type of formating, for terraform i prefer snake_case
```suggestion
variable "psql_subnet_cidr" {
```